_includes/schemas/shipments_request.json

```diff
@@ -257,10 +257,16 @@
     "label": {
       "type": "object",
       "properties": {
+        "format": {
+          "type": "string",
+          "enum": ["pdf_a5", "pdf_a6"],
```

Comment:
Wie ist das mit den ZPL Werten die wir hier erlauben? Das betrifft zwar nur wenige Kunden, frage mich aber ob das trotzdem mit in das Schema müsste?